doc: document Android CI artifact architecture

jarolrod opened the pull request:
Documents that the Android CI artifact is for ARM64 android machines.

shaavan commented on src/qml/README.md:
```diff
@@ -7,7 +7,7 @@ This directory contains the source code for an experimental Bitcoin Core graphic
 - for Windows: [`insecure_win_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip)
 - for macOS: [`insecure_mac_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip)
-- for Android: [`insecure_android_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip)
+- for ARM64 Android: [`insecure_android_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip)
```
Is there a way to reconfigure the URL and the corresponding CI artifact name so that:
`insecure_android_apk.zip` is replaced by `insecure_android_arm64_apk.zip`.

```suggestion
- for ARM64 Android: [`insecure_android_arm64_apk.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_arm64_apk.zip)
```

If possible, we should make this change, as this will make it more straightforward for the user to understand the artifact's intended target architecture.

jarolrod replied:
I think this is fine as is until we are to support x86 android

hebasto commented:
@jarolrod 

Rebase is required :tiger2:

jarolrod commented:
updated from [e30558c ](https://github.com/bitcoin-core/gui-qml/commit/e30558c00040b6d13a07729afef2476c0c4c3b69)to [3d1312c](https://github.com/bitcoin-core/gui-qml/commit/3d1312c175a24e9aeb1ce67fee5b71496aa27798)

changes: rebased over changes on master

shaavan reviewed (approved):
ACK 3d1312c175a24e9aeb1ce67fee5b71496aa27798

- Verified that the link works correctly.
